docs/REST API/rest-api.md

```diff
@@ -3663,6 +3665,114 @@ OS vulnerability data is currently available for Windows and macOS. For other pl
 
 ```
 
+### Get host's software
+
+`GET /api/v1/fleet/hosts/:id/software/titles`
+
+#### Parameters
+
+| Name | Type    | In   | Description                  |
+| ---- | ------- | ---- | ---------------------------- |
+| id   | integer | path | **Required**. The host's ID. |
+| page | integer | query | Page number of the results to fetch.|
+| per_page | integer | query | Results per page.|
+
+#### Example
+
+`GET /api/v1/fleet/hosts/123/software/titles`
+
+##### Default response
+
+`Status: 200`
+
+```json
+{
+  "software_titles": [
+    {
+      "id": 121,
+      "name": "Google Chrome.app",
+      "software_package": "GoogleChrome.pkg",
```

Comment:
```suggestion
      "package_available_for_install": "GoogleChrome.pkg",
```
Maybe we should lean towards being explicit here?

As an API user I want to know what I can do w/ this package. I'm not sure it's obvious w/o the word "install".